.circleci/config.yml

```diff
@@ -5,30 +5,41 @@ orbs:
   azure-cli: circleci/azure-cli@1.1.0
 
 jobs:
-  unit:
-    docker:
-      - image: dataders/pyodbc:1.4
+  build:
```

Comment:
Given that we only pip install some packages, the build job is maybe not needed anymore. What could be nice, is using the python orb like [here](https://github.com/CircleCI-Public/python-orb/issues/70#issuecomment-788144946).

add to top level:

```
  orbs:
    python: circleci/python@1.3
```

And use the [python/install-packages](https://circleci.com/developer/orbs/orb/circleci/python#commands-install-packages) command to install the packages, for example:

```
        - python/install-packages:
            pkg-manager: pip
            args: tox
            pypi-cache: true
            venv-cache: false
            pre-install-steps:
              - run: pyenv install 3.7.8
              - run: pyenv install 2.7.18
              - run: pyenv local 3.8.7 3.7.8 2.7.18
```

This could be the first step of each job (change the installs when only `dbt` is needed) and with caching it would probably be fast. However, a build job would also be ok, but I would try to avoid the virtual env.

Reply:
I added back the `python/install-packages` command. but it can't work to replace `pip install .` AFAIK. regardless, could be a future PR if need be.